packages/generator/test/template-helpers-get-base-uri.test.ts

```diff
@@ -7,27 +7,25 @@
 "use strict";
 
 import { expect } from "chai";
-import * as lib from "webapi-parser";
-const wap = lib.WebApiParser;
-const domain = lib.model.domain;
+import { default as amf, model } from "amf-client-js";
 
 import { getBaseUri } from "../src/templateHelpers";
 
-describe("Test getBaseUri template help function", () => {
-  before(() => wap.init());
+amf.plugins.document.WebApi.register();
 
+describe("Test getBaseUri template help function", () => {
```

Comment:
```suggestion
describe("getBaseUri template helper function", () => {
```
Including "test" in a test description is redundant.